analyzers/tests/SonarAnalyzer.CSharp.Styling.Test/Rules/FileScopeNamespaceTest.cs

```diff
@@ -23,7 +23,13 @@ namespace SonarAnalyzer.CSharp.Styling.Test.Rules;
 [TestClass]
 public class FileScopeNamespaceTest
 {
+    private readonly VerifierBuilder builder = new VerifierBuilder<FileScopeNamespace>().WithOptions(ParseOptionsHelper.CSharpLatest).WithConcurrentAnalysis(false);
+
     [TestMethod]
     public void FileScopeNamespace() =>
-        new VerifierBuilder<FileScopeNamespace>().WithOptions(ParseOptionsHelper.CSharpLatest).AddPaths("FileScopeNamespace.cs").Verify();
+        builder.AddPaths("FileScopeNamespace.cs").Verify();
+
+    [TestMethod]
+    public void FileScopeNamespace_Compliant() =>
+        builder.AddPaths("FileScopeNamespace.Compliant.cs").Verify();
```

Comment:
For this kind of test, I believe it is better to be explicit that there are no issues at all:
```suggestion
        builder.AddPaths("FileScopeNamespace.Compliant.cs").VerifyNoIssueReported();
```

In my opinion, `Verify` should fail if there is no issue reported.